db/seeds.d/020-provisioning_templates_list.rb

```diff
@@ -1,5 +1,21 @@
 class ProvisioningTemplatesList
   class << self
+    def template_description
+      @template_description ||=
+      {
+        "PXELinux" => "Deployed to the TFTP server to ensure the Host boots the correct installer using pxelinux.0 file",
```

Comment:
Currently I'm not translating the help text at db level, but only at the view level. So, after running `rake gettext:find` task, I'm unable to see the translations in the foreman.po file. Am I missing something?
@ares @lzap your comments?